src/mscorlib/model.xml

```diff
@@ -1051,6 +1051,9 @@
     </Type>
     <Type Name="System.Runtime.Loader.AssemblyLoadContext" condition="FEATURE_HOST_ASSEMBLY_RESOLVER">
       <Member Name="#ctor" />
+      <Member Name="#ctor(System.Boolean)" />
+      <Member Name="Finalize" />
+      <Member MemberType="Property" Name="IsCollectible" />
```

Comment:
yes there are still lots of places where FEATURE_COLLECTIBLE_ALC is not setup. Didn't want to introduce too early in the process, as it is going to make the code sometimes a lot less readable.